package.json

```diff
@@ -73,8 +73,8 @@
     "autoprefixer": "10.4.13"
   },
   "engines": {
-    "npm": ">=7.0.0",
-    "node": ">=16.0.0"
+    "npm": ">=9.0.0",
+    "node": ">=18.0.0"
```

Comment:
Looks like the tests are not being run anymore
![image](https://user-images.githubusercontent.com/9084735/230115479-50ca3baa-66ef-41f4-bf55-5c9ea2af2121.png)

Probably will nee to update dockerfile and publish new image. (or we could leave it as node 16 engine in package.json but keep 18 in contributing.md)

Reply:
ahh, its docker change :O
And i wanted to collect easy contribution :D
well ok, what you want me to do?

Reply:
Either: update node version in dockerfile and publish new images (could be done in a separate PR)

Or: revert the package.json change, and leave the contributing.md change.

Take your pick :)

Reply:
I'd suggest the second option:
> revert the package.json change, and leave the contributing.md change.

Reply:
done, but ideally, would be cool to have here also correct versions, if node 16 fails (errors of those usually are unclear whereas having engine set correctly would deliver understandable error)